with-realm/index.js

```diff
@@ -0,0 +1,13 @@
+import 'expo-dev-client';
```

Comment:
Drop one of these imports. The dev client config plugin is a bit flakey for this